### ISSUE TITLE
scientific extension: rel of cite-as links for sci:doi and for pub dois

### ISSUE THREAD

**Issue by schwehr** [radiantearth/stac-spec#915]
This came up here https://github.com/stac-utils/pystac/pull/199#discussion_r516974464 where we are trying to implement the scientific extension in pystac.

It's difficult to distinguish the sci:doi link from the publication doi links.  It feels weird that there is no way to distinguish the two types of links without lookin in the sci:doi field and pubs.  What if the data doi and the publication doi are the same?  What is someone adds a cite-as on their own?

```javascript
{
  "sci:doi": "10.5061/dryad.s2v81.2",
  "sci:citation": "Vega GC, Pertierra LR, Olalla-Tárraga [SNIP]",
  "sci:publications": [
    {"doi": "10.1038/sdata.2017.78", "citation": "Vega GC, Pertierra LR, Olalla-Tárraga [SNIP]"}
  ],
  "links": [
    {"rel": "cite-as", "href": "https://doi.org/10.5061/dryad.s2v81.2"},
    {"rel": "cite-as", "href": "https://doi.org/10.1038/sdata.2017.78"}
  ]
}
```

**Comment by m-mohr** [comment]
The links were just intended as simplification for clients and users, mostly for the ones not supporting the scientific extension so that user could still follow links. It was just meant as a list of related dois to follow, not to transport any additional information or so.
If two links would be the same, there should be just one link to it. The thing is that the DOI links can easily be generated if the client understands the sci extentsion/dois anyway so those clients don't really need the links.

**Comment by lossyrob** [comment]
The issue here isn't when the links are the same - it's when the links are different, and they are both labeled as 'cite-as'. 'cite-as' to me indicates - if you are going to cite this data, use this DOI. Which makes sense for the main "doi" field of the scientific extension.

However, for the publications, which are a "List of relevant publications referencing and describing the data.", the DOIs are _not_ what users of the data should be citing. Therefore the "cite-as" link seems to be a misnomer here. The IEFT draft linked to in the relation type description in the extension spec is titled "cite-as: A Link Relation to Convey a Preferred URI for Referencing" - which the publication DOI links are not.

I'd recommend that the rel-type of the publication DOIs be changed from 'cite-as' to something else - perhaps 'publication-doi'? Happy to make a PR (in time) if folks agree this is a good change.

**Comment by m-mohr** [comment]
Well, from my understanding the links are the preferred URIs for Referencing the publications. I had seen the links as addition to the individual sci:... entries and be used in that context, not just in the context of the metadata file. But your thinking also makes sense to me, but then it should be ensure that there's always a maximum of one cite-as, right? So either a new rel type would work or put the links into the publication object...

**Comment by lossyrob** [comment]
I think links in the publication objects would be odd, since it seems like "links" are the place for those. A new rel type would work, but another option is that we just have one `cite-as` and we don't give links to the referencing publications, because as you mentioned this is a convenience as users can figure that out for themselves, and the most important link is the cite-as.

Which option to people prefer? No links for publications, or links but with a new relation type? If the latter, does `publication` work, or are there other suggestions?

**Comment by m-mohr** [comment]
I think both options are fine for me... Maybe go for simplicity and no links?

**Comment by matthewhanson** [comment]
I prefer the putting links in `links` and using specific rel types to define what it is.  Additional `sci` fields in an extension would be used to provide additional context if needed.

**Comment by lossyrob** [comment]
Closed with #931
